models/surveys.go

```diff
@@ -418,6 +435,26 @@ func (api *API) getSurveyID(surveyID string) error {
 	return api.GetSurveyIDStmt.QueryRow(surveyID).Scan(&id)
 }
 
+func isUuid(id string, idName string, w http.ResponseWriter) bool {
+	_, err := uuid.Parse(id)
+
+	if err != nil {
+		re := NewRESTError("400", "Invalid "+idName+" ID")
```

Comment:
When using http codes, should always use the `http` package constants (i.e. `http.StatusBadRequest`)